src/m_menu.c

```diff
@@ -4115,23 +4110,16 @@ static const char *menu_background_strings[] = {
 
 setup_menu_t gen_settings2[] = { // General Settings screen2
 
-  {"Mouse Settings"     ,S_SKIP|S_TITLE, m_null, M_X, M_Y},
-
-  // [FG] double click acts as "use"
-  {"Double Click acts as \"Use\"", S_YESNO, m_null, M_X,
-   M_Y+ gen2_mouse1*M_SPC, {"dclick_use"}},
-
-  {"Permanent Mouselook", S_YESNO, m_null, M_X,
-   M_Y+ gen2_mouse2*M_SPC, {"mouselook"}, 0, M_UpdateMouseLook},
+  {"Display Options"  ,S_SKIP|S_TITLE, m_null, M_X,
+   M_Y + gen2_title1*M_SPC},
 
-  // [FG] invert vertical axis
-  {"Invert vertical axis", S_YESNO, m_null, M_X,
-   M_Y+ gen2_mouse3*M_SPC, {"mouse_y_invert"}},
+  {"Enable predefined translucency", S_YESNO|S_STRICT, m_null, M_X,
+   M_Y+ gen2_trans*M_SPC, {"translucency"}, 0, M_Trans},
 
-  {"", S_SKIP, m_null, M_X, M_Y + gen2_end1*M_SPC},
+  {"Translucency filter percentage", S_NUM, m_null, M_X,
+   M_Y+ gen2_transpct*M_SPC, {"tran_filter_pct"}, 0, M_Trans},
 
-  {"Display Options"  ,S_SKIP|S_TITLE, m_null, M_X,
-   M_Y + gen2_title2*M_SPC},
+  {"", S_SKIP, m_null, M_X, M_Y + gen2_stub2*M_SPC},
```

Comment:
Shouldn't this be `gen2_stub1`?

Reply:
Thanks, fixed in https://github.com/fabiangreffrath/woof/commit/9180d56ec9aded63ccab3028165d7c32a5e3c423